website/scripts/ros/ros_host_vehicle_cmd.js

```diff
@@ -65,3 +65,60 @@ function subscribeToVehicleCMD()
         }
     });
 }
+
+/**
+ * /hardware_interface/speed_pedals
+ */
+function subscribeToSpeedPedals()
+{
+    var listener = new ROSLIB.Topic({
+        ros: g_ros,
+        name: "/hardware_interface/speed_pedals",
+        messageType: "automotive_platform_msgs/SpeedPedals"
+    });
+    listener.subscribe(function(message)
+    {
+        //Check ROSBridge connection before subscribe a topic
+        IsROSBridgeConnected();
+        if(message!=null && message.brake !=null)
+        {
+              //Brake(0-1 percent)
+              if(message.brake != null && g_brakeCircle != null)
+              {
+                let max, brakeLimit = '';
+                //set brake  limit to the host vehicle deceleration(brake) limit if exist in session, otherwise default to 6
+                if(message.brake>0)
+                    brakeLimit = session_hostVehicle.brakeLimit;
+                  
+                max =  brakeLimit != null && brakeLimit.length > 0? brakeLimit: 6;
+                let value = message.brake * max; //brake value
+                updateBrake(g_brakeCircle,max,value);
+              }
+        }
```

Comment:
The contents of this function don't make much sense to me. Why are we evaluating the deceleration limit of the vehicle when trying to display the percentage brake application? It seems like this function should be a 1-1 pass through of the percentage to the display.

Reply:
The deceleration limit is different among cars. The limit is the parameters I get from the configuration file.

Reply:
Yes, I assume the brake range is 0-1. Depends on the different cars' setting the max brake limit changes. Based on the max limit, the displayed value is within range 0-max.

Reply:
The braking report is a percentage, but if you are trying to show deceleration then you can't use the max deceleration parameter because that is only a target not the actual. Its always possible for the vehicle to exceed that but it will try not to.

Reply:
Addressed https://github.com/usdot-fhwa-stol/carma-web-ui/commit/c1c713087510423612dd010d0802e4e1bdebe1b3